docs/modules/ROOT/pages/guide-keep3r.adoc

```diff
@@ -0,0 +1,138 @@
+# Setting up a keep3r
+
+This guide will walk you through the process of running a Keeper in the https://keep3r.network/[keep3r.network] by leveraging Defender xref:relay.adoc[Relays] and xref:autotasks.adoc[Autotasks].
+
+WARNING: The keep3r.network project is in beta. Use at your own risk.
+
+[[what-is-a-keeper]]
+## What is a Keeper?
+
+In the context of the keep3r network, Keepers are participants who execute available jobs in exchange for a reward. A contracts may register a job to be executed, and once it's available, a registered Keeper can send a transaction to execute it.
+
+[quote, 'https://docs.keep3r.network/#keepers[keep3r.network documentation]']
+____
+A Keeper is the term used to refer to an external person and/or team that executes a job. This can be as simplistic as calling a transaction, or as complex as requiring extensive off-chain logic. The scope of Keep3r network is not to manage these jobs themselves, but to allow contracts to register as jobs for keepers, and keepers to register themselves as available to perform jobs. It is up to the individual keeper to set up their devops and infrastructure and create their own rules based on what transactions they deem profitable.
+____
+
+[[prerequsites]]
+## Prerequisites
+
+To get started, you'll need the following:
+
+. *A Defender mainnet account*. Head over to https://defender.openzeppelin.com/[Defender] to sign up for a new account, and then https://openzeppelin.com/apply/[apply for mainnet access] indicating that you are interested in running a keep3r. We'll contact you shortly to activate your account for mainnet access. Note that mainnet accounts are free for now, but will be paid starting end of year.
```

Comment:
Do we have a specific time frame for when accounts will be chargeable?

Reply:
December/January, still TBD